### ISSUE TITLE
dungeon problem

### ISSUE THREAD

**Issue by Nightshade3l** [DexterHuang/CyberCodeOnline#1166]
when entering a dungeon, it is on the "loading dungeon" screen and nothing happens, restarting does not work

**Comment by DexterHuang** [comment]
are you able to click leave dungeon?

**Comment by Nightshade3l** [comment]
I can safely leave the dungeon, but upon entering the screen is black
again, and the game is super!

пт, 3 Дек 2021 г., 12:10 DexterHuang ***@***.***>:

**Comment by DexterHuang** [comment]
Have dungeon worked for you before? What phone are you using?

**Comment by Nightshade3l** [comment]
Samsung J7 phone, I downloaded the game only yesterday, and as I moved to
level 4, I tried to enter by invitation, but for some reason it does not
enter. Why could this be?

пт, 3 Дек 2021 г., 12:50 DexterHuang ***@***.***>:

**Comment by DexterHuang** [comment]
May I know what's your android version?
You may want to try updating your Android Webview and Google chrome from playstore

**Comment by Nightshade3l** [comment]
yeah, I checked it too, android marshmallow 6. software at the level.

пт, 3 Дек 2021 г., 12:54 DexterHuang ***@***.***>:

**Comment by Nightshade3l** [comment]
By the way, the guy in the chat just complained that he could not go to the
dungeon by invitation

пт, 3 Дек 2021 г., 12:10 DexterHuang ***@***.***>:

**Comment by DexterHuang** [comment]
could you try update android webview and google chrome from store? if you have system update you can try that too :)

**Comment by Mancunian1945** [comment]
I got a problem at dungeon too..its not "freeze" or "black screen". But I don't see and cant move my character to explore the dungeon
![Screenshot_20220130-211838](https://user-images.githubusercontent.com/98684744/151703732-c085b8fd-2465-402e-9a79-75ce3ffc3cdb.png)
What should I do?
